migrations/20181023144034_created_introduction_columns.js

```diff
@@ -0,0 +1,21 @@
+exports.up = async function(knex, Promise) {
+  await knex.schema.table("Sections", table => {
+    table.string("introductionTitle");
+    table.string("introductionContent");
+    table.boolean("introductionEnabled").defaultsTo(false);
+  });
+  await knex.raw(`DROP VIEW "SectionsView"`);
+  return knex.raw(`
+    CREATE OR REPLACE VIEW "SectionsView" AS (
+      SELECT *, ROW_NUMBER () OVER (
+        PARTITION BY "questionnaireId"
+        ORDER BY "order" ASC
+      ) - 1 AS "position"
+      FROM "Sections"
+      WHERE "isDeleted" = false
+    )`);
+};
+
+exports.down = async function(knex, Promise) {
+  return Promise.resolve();
```

Comment:
Minor thing but, I think the current method for dealing with this down is to return lodash's noop function.